docker-compose.ci.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+
+services:
+  app:
+    build: .
+    environment:
+      DB_HOST: localhost
```

Comment:
Since the app is also running in a container, I believe the `DB_HOST` should be set to `pg` (the name of the PostgreSQL service defined in `docker-compose.yml`.

Reply:
While testing, I've found that `DB_HOST `value of pg leads to failure, while a value of dev-directory_pg_1 gets accepted. I'm unable to explain this. Would you know why this might be the case?